src/style.css

```
@@ -1 +1,69 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars--0 {
+  display: flex;
+}
+
+.stars--1 {
+  display: flex;
+}
+
+.stars--2 {
+  display: flex;
+}
+
+.stars--3 {
+  display: flex;
+}
+
+.stars--4 {
+  display: flex;
+}
+
+.stars--5 {
+  display: flex;
+}
+
+.darkstars {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.goldstars {
+  background-image: url(images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--0 :last-child {
+  margin: 0;
+}
+
+.stars--1 :last-child {
+  margin: 0;
+}
+
+.stars--2 :last-child {
+  margin: 0;
+}
+
+.stars--3 :last-child {
+  margin: 0;
+}
+
+.stars--4 :last-child {
+  margin: 0;
+}
+
+.stars--5 :last-child {
+  margin: 0;
+}
```

```suggestion
.stars__star:last-child {
  margin-right: 0;
```
it can be shorter as every star will have same class